test/verify/check-docker

```diff
@@ -339,6 +339,16 @@ CMD ["/bin/sh"]
         b.wait_present("#container-terminal .console-ct .terminal")
 
         b.focus('#container-terminal .console-ct .terminal')
+
+        # Wait for the container to wake up
+        try:
+            b.key_press( [ 'Return', 'Return', 'Return' ] )
+            b.wait_in_text("#container-terminal", "#")
+        except Error, ex:
+            if not ex.msg.startswith('timeout'):
+                raise
```

Comment:
Is sending three returns at once enough, or does this need to happen in a loop? Below it's calling the same `wait_in_text` again, which is now redundant.